python_ta/checkers/forbidden_io_function_checker.py

```diff
@@ -68,6 +69,13 @@ def visit_call(self, node):
                 ):
                     if name in self.linter.config.forbidden_io_functions:
                         self.add_message("forbidden-IO-function", node=node, args=name)
+                elif (
+                    isinstance(scope_parent, type(None))
```

Comment:
This isn't necessary, it's enough to check `isinstance(scope, nodes.Module)`

Reply:
Makes sense! Addressed.